techlibs/xilinx/xc7_xcu_brams.txt

```diff
@@ -73,33 +73,36 @@ bram $__XILINX_RAMB18_TDP
 endbram
 
 match $__XILINX_RAMB36_SDP
-  min bits 4096
+  min bits 1024
   min efficiency 5
   shuffle_enable B
   make_transp
   or_next_if_better
 endmatch
 
 match $__XILINX_RAMB18_SDP
-  min bits 4096
+  min bits 1024
   min efficiency 5
   shuffle_enable B
   make_transp
   or_next_if_better
 endmatch
 
 match $__XILINX_RAMB36_TDP
-  min bits 4096
+  min bits 1024
   min efficiency 5
   shuffle_enable B
   make_transp
   or_next_if_better
 endmatch
 
 match $__XILINX_RAMB18_TDP
-  min bits 4096
+  min bits 1024
   min efficiency 5
   shuffle_enable B
   make_transp
 endmatch
 
+# [[CITE]] 7 Series FPGAs Memory Resources User Guide (UG473),
```

Comment:
Can you move this to before the first `match` block, and state that the `min bits` value was taken from this document (with a link) please?